src/app/stacktrace_win.h

```diff
@@ -108,6 +112,68 @@ BOOL CALLBACK straceWin::EnumModulesCB(LPCSTR ModuleName, DWORD64 BaseOfDll, PVO
 }
 
 
+/**
+* Cuts off leading 'dir' path from 'file' path, otherwise leaves it unchanged
+* returns true iff 'dir' is an ancestor of 'file'
+*/
+bool straceWin::makeRelativePath(const QString& dir, QString& file)
+{
+    auto d = QDir::toNativeSeparators(QDir(dir).canonicalPath());
+    auto f = QDir::toNativeSeparators(QFileInfo(file).canonicalFilePath());
+
+    // append separator at the end of dir
+    auto separator = QDir::separator();
+    if (!d.isEmpty() && (d[d.length() - 1] != separator))
+        d += separator;
+
+    if (f.startsWith(d, Qt::CaseInsensitive)) {
+        f.remove(0, d.length());
+        file.swap(f);
+
+        return true;
+    }
+
+    return false;
+}
+
+QString straceWin::getSourcePathAndLineNumber(HANDLE hProcess, DWORD64 addr)
+{
+    IMAGEHLP_LINE64 line{};
```

Comment:
maybe it just me, but I prefer old school `IMAGEHLP_LINE64 line = {0};`
don't change it yet, wait for others comment.